### PR TITLE
Add QimaiSearcher

### PR DESCRIPTION
**Pull request by ryu-s-r**
Add new searcher for Qimai

### CODE REVIEW

File: QimaiSearcher/QimaiSearcher.xml [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+        	url="https://www.qimai.cn/search/android/search/%TARGET%" 
+        	label="@search_by_packagename"
+        	icon="QimaiIcon.png"
+        	pluginGroup=".searcherGroup"
+        	description="@search_by_packagename_desc" />
+
+        <package-searcher-linker
+        	target="label"
+        	url="https://www.qimai.cn/search/android/search/%TARGET%"
+        	preferLanguage="ko;en"
```
Comment (sunggyu-kam) [inline QimaiSearcher/QimaiSearcher.xml]: The preferLanguage is used to get the label value from the app resource.
It is recommended to set the below settings for Chinese-first search in Chinese stores.

preferLanguage="zh-rCN;zh;en"

**Review by sunggyu-kam — COMMENTED** [review]
I recommend to change the preferLanguage.

File: QimaiSearcher/QimaiSearcher.xml (hunk 1 above)
Comment (ryu-s-r) [reply QimaiSearcher/QimaiSearcher.xml]: Thank you. Just fixed.

**Comment by sunggyu-kam** [comment]
Thank you for your commit.

**Pull request merged**